bluephos/tasks/filter_pipeline.py

```diff
@@ -0,0 +1,61 @@
+import pandas as pd
+from dplutils.pipeline import PipelineTask
+from bluephos.modules.filter import FilterTask
+
+
+# Helper function to create both filter_in and filter_out tasks
+def filter(df: pd.DataFrame, column: str, threshold: float, filter_in=True) -> pd.DataFrame:
+    filter_task = FilterTask(column_name=column, threshold=threshold, filter_in=filter_in)
```

Comment:
I mean here, I don't see much a reason to wrap a instance just to throw away. The logic of `FilterTask.task` could just go inline here.

Reply:
I agree, using a class for the filter isn’t necessary right now. (I initially considered it for more dynamic filtering, but it's not needed at this stage). Thanks for your input.